kube-fluentd/gcp/fluent.conf

```diff
@@ -0,0 +1,8 @@
+# This is the root config file, which only includes components of the actual configuration
+
+# Do not collect fluentd's own logs to avoid infinite loops.
+<match fluent.**>
+  type null
+</match>
+
+@include /etc/fluent/config/*.conf
```

Comment:
I intentionally left it this way, because for now we want to able to tweak these parameters, including in tests, as they influence the stability of fluentd container

This part will be removed from the configuration with switching to the Stackdriver container image.